locustfile.py

```diff
@@ -0,0 +1,26 @@
+from locust import HttpUser, task, between
+
+
+class ChatUser(HttpUser):
+    wait_time = between(60,61)
+
+    def on_start(self):
```

Comment:
This should just be a task, otherwise there won't be a wait period between this `.get()` and the subsequence `.post()` in the next function